recipes-kernel/linux/files/5.x-kernel/0008-feat-change-mmc-order-using-alias-in-dts.patch

```diff
@@ -1,21 +1,38 @@
-From 5df72a0aa9663094988a20fb731c07680b931a97 Mon Sep 17 00:00:00 2001
-From: Su Baocheng <baocheng.su@siemens.com>
-Date: Tue, 22 Dec 2020 15:08:17 +0800
-Subject: [PATCH 12/14] feat:change mmc order using alias in dts
+From 6f9df25490ac6e69b75ddbdd59b9824a46b508a1 Mon Sep 17 00:00:00 2001
+From: "le.jin" <le.jin@siemens.com>
+Date: Wed, 9 Oct 2019 15:22:09 +0800
+Subject: [PATCH 08/14] feat:change mmc order using alias in dts
 
 1. modify kernel to support mmc alias in dts
 2. change SD to mmc0 and EMMC to mmc1 via alias in dts
 
```

Comment:
Unfortunately, it's still unclear if this can be accepted upstream. I will into that again - we get no answer from speculating, we need to propose/discuss with a strong story.

Reply:
Actually I suggest it better to using “SD” to represent sd，and “emmc” to represent emmc insead of using mmc0 and mmc1 in our board.

using “SD ” and “ecmm” could display information directly  I think

Reply:
DT node names are defined in the upstream DT - would need changes there first. But who is evaluating them outside of the DT at all?